luigi/server.py

```diff
@@ -68,6 +68,26 @@ def initialize(self, scheduler):
         self._scheduler = scheduler
 
     def get(self, method):
+        if method not in [
+            'add_task',
+            'add_worker',
+            'dep_graph',
+            'disable_worker',
+            'fetch_error',
+            'get_work',
+            'graph',
+            'inverse_dep_graph',
+            'ping',
+            'prune',
```

Comment:
To clarify. I'm not sure it makes sense for clients to be able to call `prune`. Many other of the API calls does call `prune` internally themselves, like `graph`, `inverse_dep_graph` etc.

This was just a minor comment so I just merged right away.